### DISCUSSION TITLE
InputMask don't overwrite value

### DISCUSSION THREAD

**Discussion by luccotta** [primefaces/community#2726] [PrimeFaces] [answered]
Hello

In JSF Primefaces 13, I have the following inputMask. If it already has a value, if I put the cursor before the letter and if I type another letter, the value is not replaced. Instead, the cursor just move to after the letter, but the letter itself doesn't change. In Primefaces 8 this used to work. How can I fix it, so the value is replaced?

<p:inputMask slotChar="" maxlength="1" mask="a" value="#{myBean.letter}" />

Thanks

**Comment by melloware** [comment]
the InputMask library changed between 8 and 13. its now using https://github.com/RobinHerbots/Inputmask because the old jquery inputmask was dead and abandoned and this library is actively maintained.

So the behavior will be different in certain circumstances but it was worth the upgrade.

**Reply by luccotta** [reply]
Could you please let me know the old library GitHub?

**Reply by melloware** [reply, marked as answer]
Sure it's right here: https://github.com/digitalBush/jquery.maskedinput

It's archived and last commit 9 years ago. That is why PrimeFaces had to move to a new library.
